__mocks__/@ui5/webcomponents-react.js

```
@@ -475,12 +540,75 @@ const TableCell = ({ children, className, ...props }) => (
+        };
+        onChange(mockEvent);
+      }
+    };
+
+    // Add another test button to trigger edge case (null row-key)
+    const handleTestChangeWithNull = () => {
+      if (onChange) {
+        const mockEvent = {
+          target: {
+            getSelectedRows: () => [
+              { getAttribute: (attr) => null }, // This will trigger the || '' fallback
```

ESLint

[error] 'attr' is defined but never used.

(@typescript-eslint/no-unused-vars)